packages/terra-search-field/src/SearchField.jsx

```diff
@@ -250,8 +263,13 @@ class SearchField extends React.Component {
       customProps.className,
     );
 
-    const inputText = inputAttributes && Object.prototype.hasOwnProperty.call(inputAttributes, 'aria-label') ? inputAttributes['aria-label'] : intl.formatMessage({ id: 'Terra.searchField.search' });
+    const labelClassNames = cx([
+      'label',
+      theme.className,
+    ]);
 
+    const inputAriaLabelText = inputAttributes && Object.prototype.hasOwnProperty.call(inputAttributes, 'aria-label') ? inputAttributes['aria-label'] : intl.formatMessage({ id: 'Terra.searchField.search' });
```

Comment:
Do we need a hasOwnProperty method call?  Won't accessing the property just return undefined and give us the desired behavior?